spec/facility_spec.rb

```diff
@@ -2,8 +2,23 @@
 
 RSpec.describe Facility do
   before(:each) do
-    @facility = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+    @facility_new = Facility.new({name: nil, address: nil, phone: nil})
+    @facility   = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014'})
+    @facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014'})
```

Comment:
Not sure why we have two facilities with the same information, usually we don't want to do that, but wondering if you had a purpose in mind.